.github/workflows/commit.yml

```diff
@@ -11,28 +11,9 @@ jobs:
     steps:
       - name: checkout
         uses: actions/checkout@v2
-      - name: Verify Changed files
```

Comment:
I'm not super familiar with what this is actually doing, but what's the reason for it to be removed?

Reply:
James removed that in the pairing call since it wasn't actually lookin for changed files in the commit but rather files changes during the build IIRC.

Reply:
I believe we would still want a changed files check but one that looks at the PR level.